src/phoenix/server/api/helpers/playground_clients.py

```diff
@@ -795,7 +795,7 @@ async def chat_completion_create(
                 elif isinstance(event, anthropic_streaming.InputJsonEvent):
                     raise NotImplementedError
                 else:
-                    assert_never(event)
+                    assert_never(event)  # type: ignore
```

Comment:
There's a fix for this that @RogerHYang put out. It's the anthropic citations event. Can you cherry pick that and remove this ignore?